docs/source/deprecated.md

```diff
@@ -0,0 +1,25 @@
+# Deprecated API
+
+```{currentmodule} kasa
+```
+The page contains the documentation for the deprecated library api that only works
```

Comment:
```suggestion
The page contains the documentation for the deprecated library API that only works
```
Also, it is usually a good idea to add a newline only at sentence edges at sentence boundaries.